inc/tag.class.php

```diff
@@ -706,6 +706,10 @@ public static function showTagDropdown($params = [])
             || (!$obj->isNewItem() && !$obj->canUpdateItem())
         ;
 
+        if (is_array($params['id'])) {
+            $readOnly = false;
+        }
+
```

Comment:
This cancels the rights check just before, so a user without UPDATE rights will still be able to perform the massive action, which is undesirable.
```suggestion
```